twisted/topfiles/8760.feature

```diff
@@ -0,0 +1 @@
+Client and server TLS connections now support ChaCha20 ciphers when available from OpenSSL.
```

Comment:
Maybe this should start with 'twisted.internet.ssl.OpenSSLCertificateOptions now support as default ChaCha20` as from what I can see this is the only class using the private t.internet._sslverify.defaultCiphers